### PR TITLE
Switch camera feeds and vision modes

### PR DESCRIPTION
**Pull request by schineaj23**

### CODE REVIEW

**Comment by schineaj23** [comment]
This last commit should not be merged. This was here simply to get my code saved before leaving today

**Comment by schineaj23** [comment]
Well, it wouldn't break the build but it just doesn't do anything

File: src/main/java/frc/robot/subsystems/Limelight.java [hunk 1]
```diff
@@ -19,11 +25,27 @@ public void periodic() {
     final double x = tx.getDouble(0.0);
     final double y = ty.getDouble(0.0);
     final double area = ta.getDouble(0.0);
+    final String stream, cam;
+
+    if(getStream() == Constants.StreamMode.PIP_MAIN)
+      stream = "Main";
+    else
+      stream = "Secondary";
```
Comment (jschoolcraft) [inline src/main/java/frc/robot/subsystems/Limelight.java]: Even though not strictly necessary let's add `{` `}`, makes it easier to extend going forward.

**Review by jschoolcraft — APPROVED** [review]
Looks good!

Going forward I'd split this into 2 PRs

- Color sensor stuff
- Limelight stuff

**Review by schineaj23 — COMMENTED** [review]
This is my code, I need to stash them though and put in another pr

**Comment by schineaj23** [comment]
I think the best course of action is to merge then remove color sensor code and put into separate request?

**Comment by schineaj23** [comment]
Finally merging. Made new pr

**Pull request merged**